ccm_web/server/src/api/api.interfaces.ts

```diff
@@ -13,7 +13,29 @@ export interface APIErrorData {
   statusCode: number
   message: string
 }
+export interface CanvasSectionBase {
+  id: number
+  name: string
+}
+export interface createSectionError {
+  failedInput: string
```

Comment:
@ssciolla after you are convinced that this structure is good, I will change it with other API endpoints.